master/pkg/model/storage_config.go

```diff
@@ -138,9 +139,26 @@ type SharedFSConfig struct {
 
```

Comment:
blocking: don't we need to remove the `container_path` in `type SharedFSConfig` above?

Reply:
Not right now, I think.  AFAICT, that would cause paused experiments to break on upgrades.

This is a good point though... I'll make sure that we don't emit that value, and then after a couple versions it should be pretty safe to not support it.

Reply:
Can we track this in a ticket so we don’t forget?

Reply:
https://determinedai.atlassian.net/browse/DET-3104